nano/core_test/block_store.cpp

```diff
@@ -352,7 +352,7 @@ TEST (bootstrap, simple)
 	nano::logger_mt logger;
 	auto store = nano::make_store (logger, nano::unique_path (), nano::dev::constants);
 	ASSERT_TRUE (!store->init_error ());
-	auto block1 (std::make_shared<nano::send_block> (0, 1, 2, nano::keypair ().prv, 4, 5));
+	std::shared_ptr<nano::block> block1 = std::make_shared<nano::send_block> (0, 1, 2, nano::keypair ().prv, 4, 5);
```

Comment:
This clean up looks good, but since you touched the lines anyway, why not `auto block = std::make_shared...`? Not really important but just in the spirit of `if cleaning it up, clean it all the way` :D.

Reply:
This one specifically is because std::shared_ptr<nano::send_block> won't implicitly convert to unchecked_info when passed to unchecked_store::put. It won't do an upcast and also an implicit conversion.